Sources/TuistKit/Commands/TuistCommand.swift

```diff
@@ -0,0 +1,40 @@
+import Foundation
+import ArgumentParser
+import TuistSupport
+
+public struct TuistCommand: ParsableCommand {
+    public init() { }
+    
+    public static var configuration: CommandConfiguration {
```

Comment:
Is this required by the `ParsableCommand` protocol?

Reply:
Yes.